pennylane_lightning/lightning_qubit.py

```diff
@@ -113,9 +185,40 @@ def __init__(self, wires, *, c_dtype=np.complex128, shots=None, batch_obs=False)
             self.use_csingle = False
         else:
             raise TypeError(f"Unsupported complex Type: {c_dtype}")
-        super().__init__(wires, r_dtype=r_dtype, c_dtype=c_dtype, shots=shots)
+        super().__init__(wires, shots=shots, r_dtype=r_dtype, c_dtype=c_dtype, analytic=analytic)
         self._batch_obs = batch_obs
 
+        # Create the initial state. Internally, we store the
+        # state as an array of dimension [2]*wires.
+        self._state = self._create_basis_state(0)
+        self._pre_rotated_state = self._state
+
+    def _create_basis_state(self, index):
+        """Return a computational basis state over all wires.
+        Args:
+            index (int): integer representing the computational basis state
+        Returns:
+            array[complex]: complex array of shape ``[2]*self.num_wires``
+            representing the statevector of the basis state
+        Note: This function does not support broadcasted inputs yet.
+        """
+        state = np.zeros(2**self.num_wires, dtype=np.complex128)
+        state[index] = 1
+        state = self._asarray(state, dtype=self.C_DTYPE)
+        return self._reshape(state, [2] * self.num_wires)
```

Comment:
I assume for functions like these we need to replicate what is in default.qubit?
Probably makes more sense at a later time to move such static functions to a separate module in PL, and we can depend upon them at that time.